middleware/limiter/README.md

```diff
@@ -30,12 +30,13 @@ After you initiate your Fiber app, you can use the following possibilities:
 app.Use(limiter.New())
 
 // Or extend your config for customization
+//Duration of limiter.Config is deprecated, please use Expiration instead. 
 app.Use(limiter.New(limiter.Config{
 	Next: func(c *fiber.Ctx) bool {
 		return c.IP() == "127.0.0.1"
 	},
 	Max:          20,
-	Duration:     30 * time.Second,
+	Expiration:30 * time.Second,
```

Comment:
This is just me being nitpicky, but could you add a couple of spaces here to make the value line up with the other config properties?

Reply:
sure

Reply:
done